crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -38,13 +38,14 @@ module sui::sui_system {
     // - the change in stake across epochs can be at most +/- x%
     // - the change in the validator set across epochs can be at most x validators
     //
-    // TODO: The stake threshold should be % threshold instead of amount threshold.
     struct SystemParameters has store {
         /// Lower-bound on the amount of stake required to become a validator.
+        // TODO: use a different threshold for new validator joining vs. validator keeping their
+        // spot in the validator set.
         min_validator_stake: u64,
         /// Maximum number of validator candidates at any moment.
         /// We do not allow the number of validators in any epoch to go above this.
-        max_validator_candidate_count: u64,
```

Comment:
What's the reason to limit the number of candidates?

Reply:
This limit is on the number of active validators. Will update the comments to reflect that.